notebooks/demo/mg_property_graph.ipynb

```diff
@@ -0,0 +1,217 @@
+    "fn = 'hollywood.csv'\n",
+    "comp = '.gz'\n",
+    "\n",
+    "if not os.path.isfile(data_dir+fn):\n",
+    "    if not os.path.isfile(data_dir+fn+comp):\n",
+    "        print(f'Downloading {base_url+fn+comp} to {data_dir+fn+comp}')\n",
+    "        urllib.request.urlretrieve(base_url+fn+comp, data_dir+fn+comp)\n",
+    "    print(f'Decompressing {data_dir+fn+comp}...')\n",
+    "    os.system('gunzip '+data_dir+fn+comp)\n",
+    "    print(f'{data_dir+fn+comp} decompressed!')\n",
+    "else:\n",
+    "    print(f'Your data file, {data_dir+fn}, already exists')\n",
+    "\n",
+    "# File path, assuming Notebook directory\n",
+    "input_data_path = data_dir+fn"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Initialize multi-GPU environment\n",
+    "Before we get started, we need to setup a Dask local cluster of workers to execute our work and a client to coordinate and schedule work for that cluster. As we see below, we can initiate a cluster and client using only 3 lines of code."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cluster = LocalCUDACluster()\n",
+    "client = Client(cluster)\n",
+    "Comms.initialize(p2p=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Read the data from disk\n",
+    "cuGraph depends on cudf for data loading and the initial DataFrame creation. The CSV data file contains an edge list, which represents the connection of a vertex to another. The source to destination pairs is what is known as Coordinate Format (COO). In this test case, the data is just two columns. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Start ETL timer\n",
+    "t_start = time.time()\n",
+    "\n",
+    "# Helper function to set the reader chunk size to automatically get one partition per GPU  \n",
+    "chunksize = dask_cugraph.get_chunksize(input_data_path)\n",
+    "\n",
+    "# Multi-GPU CSV reader\n",
+    "e_list = dask_cudf.read_csv(input_data_path, chunksize = chunksize, delimiter=' ', names=['src', 'dst'], dtype=['int32', 'int32'])\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a graph"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cugraph.experimental import MGPropertyGraph\n",
+    "pG = MGPropertyGraph()\n",
+    "pG.add_edge_data(e_list,vertex_col_names=[\"src\", \"dst\"])\n",
+    "pG.get_num_edges()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "graph = cugraph.Graph(directed=False) \n",
+    "(louvain_df, modularity) = dask_cugraph.louvain(pG.extract_subgraph(create_using=graph))\n",
```

Comment:
Might be cleaner if this was just done inline:
```suggestion
    "(louvain_df, modularity) = dask_cugraph.louvain(pG.extract_subgraph(create_using=cugraph.Graph))\n",
```

Reply:
Done